pandas/core/internals/blocks.py

```diff
@@ -2076,15 +2076,16 @@ def get_values(self, dtype=None):
         return object dtype as boxed values, such as Timestamps/Timedelta
         """
         if is_object_dtype(dtype):
-            values = self.values.ravel()
-            result = self._holder(values).astype(object)
-            return result.reshape(self.values.shape)
+            return self._holder(self.values).astype(object)
```

Comment:
Can you add a comment here on why `_holder` can handle 2D?

Reply:
Will do